shared/data/surrogates.txt

```diff
@@ -0,0 +1,360 @@
+    }
+    const gaqTarget = {
+        push: function(arg) {
+            if (typeof arg === 'function') {
+                try {
+                    arg()
+                } catch(error) {}
+                return
+            }
+            if (Array.isArray(arg) === false) { return }
+            if (arg[0] === '_link' && typeof arg[1] === 'string' ) {
+                window.location.assign(arg[1])
+            }
+            if (arg[0] === '_set' && arg[1] === 'hitCallback' && typeof arg[2] === 'function' ) {
+                try {
+                    arg[2]()
+                } catch(error) {}
+                return
+            }
+        }
+    }
+    const gatTarget = {
+        _getTracker: function() { return new Proxy(trackerTarget, noopHandler) },
+        _getTrackerByName: function() { return new Proxy(trackerTarget, noopHandler) },
+        _getTrackers: noopReturnEmptyArray
+    }
+    const gaqObj = new Proxy(gaqTarget, noopHandler)
+    const gatObj = new Proxy(gatTarget, noopHandler)
+    window._gat = gatObj
+    const commandQueue = (window._gaq && Array.isArray(window._gaq)) ? window._gaq : []
+    while (commandQueue.length > 0) {
+        gaqObj.push(commandQueue.shift())
+    }
+    window._gaq = gaqObj
+})()
+
+google-analytics.com/analytics.js application/javascript
+(() => {
+    'use strict'
+    const noop = () => {}
+    const noopReturnNull = () => {
+        return null
+    }
+    const noopHandler = {
+        get: function(target, prop) {
+            return noop
+        }
+    }
+    const noopReturnArray = () => {
+        return []
+    }
+    const gaPointer = window.GoogleAnalyticsObject = (window.GoogleAnalyticsObject === void 0) ? 'ga' : window.GoogleAnalyticsObject
+    const datalayer = window.dataLayer
+    // execute callback if exists, see https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#hitCallback
+    const ga = function() {
+        const params = Array.from(arguments)
+        params.forEach((param) => {
+            if (param instanceof Object && typeof param.hitCallback === 'function') {
+                try {
+                    param.hitCallback()
+                } catch (error) {}
+            }
+        })
+    }
+    ga.answer = 42
+    ga.loaded = true
+    ga.create = function() { return new Proxy({}, noopHandler) }
+    ga.getByName = noopReturnNull
+    ga.getAll = noopReturnArray
+    ga.remove = noop
+    window[gaPointer] = ga
+    // prevent page delay, see https://developers.google.com/optimize
+    if (datalayer && datalayer.hide && typeof datalayer.hide.end === 'function') {
+        try {
+            datalayer.hide.end()
+        } catch (error) {}
+    }
+})()
+
+google-analytics.com/inpage_linkid.js application/javascript
+(() => {
+    const gaqObj = {
+        push: () => {}
+    }
+    window._gaq = (window._gaq === void 0) ? gaqObj : window._gaq
+})()
+
+google-analytics.com/cx/api.js application/javascript
```

Comment:
As far as I can tell this surrogate will never match anything (and it haven't matched anything before this PR either). ID extracted from this URL is `cx` while ID used in `https://staticcdn.duckduckgo.com/trackerblocking/v2.1/tds.json` is `api.js`. I say we drop it.

<img width="438" alt="ids" src="https://user-images.githubusercontent.com/985504/88044153-3ed11980-cb4e-11ea-922a-cb6585dd4022.png">

Reply:
We could keep them and change the mapping here https://github.com/duckduckgo/tracker-radar/blob/main/build-data/static/surrogates.json

Reply:
Good catch. For the time being, what do you think about updating these to:
google-analytics.com/api.js
doubleclick.net/ad_status.js
amazon-adsystem.com/amzn_ads.js
so that the mapping in Tracker Radar matches on them? That would make them work right away, and this change is only going out to the extension immediately. Then when we get these things into a better format (split up, checked into Tracker Radar) we can revisit and improve the logic?

Reply:
👍 that makes more sense